csrc/mmdeploy/apis/c/mmdeploy/CMakeLists.txt

```diff
@@ -38,13 +57,20 @@ foreach (TASK ${TASK_LIST})
             DESTINATION include/mmdeploy)
 endforeach ()
 
-if (MMDEPLOY_BUILD_SDK_CSHARP_API)
-    # build MMDeployExtern.dll just for csharp nuget package.
-    # no Installation for c/c++ package.
-    file(GLOB SRCS "*.c" "*.cpp")
-    add_library(MMDeployExtern SHARED ${SRCS})
-    target_compile_definitions(MMDeployExtern PRIVATE -DMMDEPLOY_API_EXPORTS=1)
-    mmdeploy_load_static(MMDeployExtern MMDeployStaticModules)
-    mmdeploy_load_dynamic(MMDeployExtern MMDeployDynamicModules)
-    target_link_libraries(MMDeployExtern PRIVATE MMDeployLibs)
+if (MMDEPLOY_BUILD_SDK_CSHARP_API OR MMDEPLOY_BUILD_SDK_MONOLITHIC)
+    add_library(mmdeploy SHARED)
```

Comment:
CMake Error at csrc/mmdeploy/apis/c/mmdeploy/CMakeLists.txt:69 (add_library):
  No SOURCES given to target: mmdeploy